terraform/main.mk

```diff
@@ -124,14 +124,14 @@ terraform.destroy-quiet: ## Destroy infrastructure without confirmation
 	@ echo "\n\033[36m[INFO] Please run: make secrets.delete or app.delete_secrets now\033[0m"
 
 terraform.output-to-ssm: ## Manual upload output.json to AWS SSM. Output.json encoded in base64.
-	@ cd $(ENV_DIR) && \
+	@ cd $(ENV_DIR)/.terraform && \
```

Comment:
@kobrikx do we also need to cd into .terraform here? (and why does it work here https://github.com/hazelops/icmk/pull/132/files#diff-d0414a57cfaec3da463ae028b8d3aa467a2f15a09677a49356ba933c58430138R89?)

Reply:
Fixed